dev/tools/gen_localizations.dart

```diff
@@ -134,12 +189,25 @@ String generateTranslationBundles() {
   // `MaterialLocalizationEn`). These implement everything that is needed by
   // GlobalMaterialLocalizations.
 
+  // We also generate one subclass for each locale with a script code (e.g.
+  // `MaterialLocalizationZhHant`). Their superclasses are the aforementioned
+  // language classes for the same locale but without a script code (e.g.
+  // `MaterialLocalizationZh`). This script subclass are defined in a separate
```

Comment:
The script subclasses are defined ... however there aren't classes in .arb files. Need to restate this point.